map-data/src/main/java/org/triplea/map/data/elements/PropertyList.java

```diff
@@ -33,11 +33,14 @@
       @BodyText private java.lang.String data;
     }
 
+    @SuppressWarnings("JavaLangClash")
```

Comment:
Thoughts on introducing a tag to override the class name?

Reply:
AFAIK tags cannot inherit tags. Is there actually a way to do that? If so, having the XML annotation to go ahead and add suppress warnings for 'unused' would be good. The lang clash is very limited, I would not expect for us to see more of it and if we introduce more lang clashes it's something that IMO we should very well fix. These tags being named after lang classes is really not ideal.

Reply:
I was more thinking of a `@CustomName("String")` tag, so the class can be called whatever it wants to but if this annotation is present it will "override" the class name to use the custom string instead